spring-cloud-sleuth-zipkin/src/test/java/org/springframework/cloud/sleuth/zipkin2/ZipkinAutoConfigurationTests.java

```diff
@@ -184,6 +189,25 @@ public void rabbitWinsWhenKafkaPresent() throws Exception {
 		this.context.close();
 	}
 
+	@Test
+	public void supportsMultipleReporters() {
```

Comment:
Can you report a span and verify that both senders and reporters were actually called?